site/docs/components/button/examples.mdx

```diff
@@ -76,6 +76,20 @@ A button with the prop `disabled={true}` will suppress all functionality.
 
 </LivePreview>
 
+<LivePreview componentName="button" exampleName="Loading" >
+
+## Loading
+
+Use the loading state for a button that indicates that an action is happening, but the user must wait for it to complete. This is useful for actions that take some time to process, such as submitting a form or loading data.
+
+A button with the prop `loading={true}` will display a spinner to indicate that the action is in progress.
+
+### Best practices
+
+**Accessibility** - Loading is a visual only state, therefore if you have additional accessibility requirements, you need to handle them manually. One way to do that is to announce a message to screen readers. This can be done by passing the `loadingAnnouncement` prop.
```

Comment:
Loading is no longer a visual only state? We should probably just mention the`loadingAnnouncement` should be used

Reply:
Done!